package.json

```diff
@@ -49,7 +49,7 @@
         "@oat-sa/tao-core-libs": "^0.2.0",
         "@oat-sa/tao-core-sdk": "^0.8.1",
         "@oat-sa/tao-core-ui": "^0.21.4",
-        "@oat-sa/tao-item-runner": "^0.3.1",
+        "@oat-sa/tao-item-runner": "^0.4.0",
```

Comment:
For development mode, to allow testing, you should temporarily target the branch in `package.json`:

`"@oat-sa/tao-item-runner": "github:oat-sa/tao-item-runner-fe#feature/TAO-9637/ACT_TTS_FE_Acces_the_data_about_APIP_the_content_and_related_media_files",`